package.json

```diff
@@ -8,7 +8,8 @@
     "start:dev": "npm run migrate:deploy && npm run build && npm run watch",
     "test-ci": "npm test",
     "test": "jest \"\\.spec\\.ts\"",
-    "test-integrations": "jest \"\\.integration\\.ts\" --runInBand",
+    "test-integrations": "jest --testPathPattern=src/database --runInBand",
```

Comment:
Well, the reason for splitting them up was to speed up the checks on PRs - since these tests use the database, too, they are slower than pure unit tests. I could tweak the path pattern so that it doesn't take in that extra test.

Reply:
since the new server integration tests are already in a designated folder, what if we switch this logic around so `utils.spec.ts` (and any other future non-integration tests that end up `src/database`) don't get run twice? something like:

```
"test-integrations": "jest \"\\.integration\\.ts\" --runInBand",
"test-server-integrations": "jest --testPathPattern=src/test/admin-server --runInBand",
```

Reply:
Well, the new files (the ones added in this PR) still have to be either `.integration.ts` or `.spec.ts` so they would be picked up by the `test-integrations` commands. OR we could update the Jest config file which I'm reluctant to do - I'd rather it is in sync with the backend template.

After experimenting with patterns (and discovering there are integration tests in `src/aws`, too), I decided to ditch the second test run and just add the new tests to the old integration tests run. The wait time for will not expand by much - it is, despite my initial concern, not as important as I thought. 

I also looked up ways of suppressing console output in the new tests - there are lots of options, from running Jest with the `--silent` option to mocking parts of the global `console` object, but left everything as is for now. https://stackoverflow.com/questions/44467657/jest-better-way-to-disable-console-inside-unit-tests